Gemfile

```diff
@@ -5,7 +5,3 @@ source 'https://rubygems.org'
 gem 'kitchen-docker', '>= 2.9'
 gem 'kitchen-inspec', '>= 1.1'
 gem 'kitchen-salt', '>= 0.6.0'
-# Latest versions of `train` cause failure when running `kitchen verify`
-# Downgrading to `3.2.0` until this is fixed upstream
-# https://github.com/inspec/train/pull/544#issuecomment-566055052
-gem 'train', '3.2.0'
```

Comment:
@noelmcloughlin Why did you remove this section?  The bug hasn't been resolved yet:

* https://github.com/inspec/train/issues/548#issuecomment-577990651

Reply:
I inadvertently dropped trying to merge masters and squash commits. I'll raise a PR to restore this now.

Reply:
https://github.com/saltstack-formulas/golang-formula/pull/34

Reply:
OK, thanks.